diff --git a/install/templates/conf/apache2/tusk_http.conf b/install/templates/conf/apache2/tusk_http.conf
index 0aa1e48..942cd6d 100644
--- a/install/templates/conf/apache2/tusk_http.conf
+++ b/install/templates/conf/apache2/tusk_http.conf
@@ -36,6 +36,7 @@ RewriteRule ^/index.html$ /home [PT]
 RewriteRule ^/mobi/$ /mobi/home [PT]
 RewriteRule ^/mobi/index.html$ /mobi/home [PT]
 RewriteRule ^/public/(.*)$ /nosession/$1 [PT]
+RewriteRule ^/dologout(.*)$ https://test-shib.tusk.tufts.edu:443/dologout$1 [R]
 RewriteRule ^/(tools|ws|public|search|service|case|quiz|assignment|gradebook|patientlog|sp|assessment|management|user|eval|ocw|admin)/(.*) /tusk/$1/$2 [PT]
 RewriteRule ^/lb$ /nosession/lb [PT]
 RedirectMatch ^/hsdb4/schedule/(.*)$ /view/schedule/$1
@@ -265,13 +266,6 @@ AliasMatch ^/chooser_icon/text(.*) /usr/local/tusk/current/graphics/icons/text.g
   AddType image/x-png  .png
 </Location>
 
-<Location /dologout>
-  SetHandler   perl-script
-  PerlCleanupHandler   Apache2::SizeLimit
-  Options +ExecCGI -Indexes
-  PerlHandler Apache::TicketRemove
-</Location>
-
 <Location /download/>
   SetHandler   perl-script
   PerlCleanupHandler   Apache2::SizeLimit

