diff --git a/install/templates/conf/apache2/tusk_base.conf b/install/templates/conf/apache2/tusk_base.conf
index 9e8bc09..6fc37ab 100644
--- a/install/templates/conf/apache2/tusk_base.conf
+++ b/install/templates/conf/apache2/tusk_base.conf
@@ -22,9 +22,8 @@ PerlSetEnv HSDB_DATABASE_NAME hsdb4
 LoadModule apreq_module    modules/mod_apreq2.so
 APREQ2_ReadLimit 1024M
 
-# TUSK Setting for apache 2 upload sizes
-LoadModule apreq_module    modules/mod_apreq2.so
-APREQ2_ReadLimit 1024M
+# Uncomment for Shibboleth SSO
+#LoadModule mod_shib /usr/lib64/shibboleth/mod_shib_22.so
 
 <Perl>
 #!/usr/bin/perl

